here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/IntHandlerForEventHandlers.java

```diff
@@ -57,4 +66,22 @@ public IntHandlerForEventHandlers(final @NotNull INaksha hub) {
       return notImplemented(event);
     }
   }
+
+  private @NotNull Result validateWriteRequest(final @NotNull WriteRequest<?, ?> wr) {
+    Result result = null;
+    for (final WriteOp<?> wOp : wr.queries) {
+      final EventHandler eventHandler = (EventHandler) wOp.feature;
+      // Common Plugin specific validations
+      result = validateWritePluginRequest(eventHandler);
+      if (result instanceof ErrorResult) return result;
+      // TODO : handler specific validations in future, as needed
+      if (!Objects.equals(eventHandler.getClassName(),"com.here.naksha.lib.handlers.DefaultStorageHandler")) {
```

Comment:
condition can be simplified as "if(eventHandler.getClassName().equals("com.here.naksha.lib.handlers.DefaultStorageHandler"))